### PR TITLE
Add docs on secrets in the clusters 🤫

### PR DESCRIPTION
**Pull request by bobcatfish**
# Changes

I tried to update some of the secrets today but I had to guess which
ones were the right ones because our docs had fallen way behind what is
actually in the cluster. There are many many secrets in these clusters
and I don't know what they all do, but I tried to make a start at
writing some of them down.

I'd really like to have a "clusters" folder which has all the config
(e.g. robocat and prow dirs) and docs for the clusters we are using, but
I have a feeling moving these around will mean our configuration needs
to be updated and I didn't feel I could take that one on today.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by bobcatfish** [comment]
thanks @vdemeester XD i think that was stuck b/c i broke the secrets briefly 😅

File: README.md [hunk 1]
```diff
@@ -2,18 +2,30 @@
...
 - [Gubernator](gubernator/README.md) is used for holding and displaying [Prow](prow/README.md) logs
 - [Boskos](boskos/README.md) is used to control a pool of GCP projects which end to end tests can run against
 - [Peribolos](tekton/resources/org-permissions/README.md) is used to control org and repo permissions
 
+## Support
+
+If you need support, reach out [in the tektoncd slack](https://github.com/tektoncd/community/blob/master/contact.md#slack)
+via the `#plumbing` channel.
+
+[Members of the Tekton governing board](goverance.md)
```
Comment (sbwsg) [inline README.md:18]: governance.md?

**Pull request merged**